src/ray/rpc/node_manager/node_manager_server.h

```diff
@@ -10,6 +10,12 @@
 namespace ray {
 namespace rpc {
 
+#define RAY_NODE_MANAGER_RPC_HANDLERS                              \
```

Comment:
Can you add the same comment that we have in `core_worker.h` about how to add a new handler?

Reply:
Maybe we can add a comment that refers to `core_worker.h`? So if we modify the comment in the future, we don't need to modify multiple files.

Reply:
makes sense. I added a note here to refer to core_worker.h